enrollment-server/src/main/java/com/wultra/app/enrollmentserver/impl/service/converter/MobileTokenConverter.java

```diff
@@ -142,11 +142,12 @@ private FormData prepareFormData(
         return formData;
     }
 
-    private static StringSubstitutor createStringSubstitutor(final Map<String, String> parameters) {
+    static StringSubstitutor createStringSubstitutor(final Map<String, String> parameters) {
         if (CollectionUtils.isEmpty(parameters)) {
             return null;
         } else {
             final Map<String, String> escapedParameters = parameters.entrySet().stream()
+                    .filter(entry -> entry.getValue() != null)
```

Comment:
@zcgandcomp I am not sure what is the goal of this issue. Should we filter out `null` values as suggested in this PR or fix NPE and include it? If so, @jandusil see https://stackoverflow.com/a/24634007/204950

Reply:
The default behavior is that if the value is omitted the key is not sent to the mobile phone. So you can "skip" the null values. 
Reason:
- They are not possible to display as of now anyway.
- The mobile app can have a problem with processing another null.

Reply:
Ok, this implementation skips `null` values.